esp/services/ws_dfu/ws_dfuService.cpp

```diff
@@ -5884,8 +5886,6 @@ void CWsDfuEx::getFilePartsInfo(IEspContext &context, IFileDescriptor &fileDesc,
     ForEach(*pi)
     {
         IPartDescriptor& part = pi->query();
-        if (isPartTLK(&part))
```

Comment:
Would we want to keep ignoring the TLK for 1.55 and earlier?

Reply:
wondering same, see my comment below

Reply:
good point